src/pages/Information/Information.jsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 const Information = () => (
   <>
     <div data-test="information-display">
-      <h2 data-test="heading-display">What is HacktoberFest?</h2>
+      <h2 data-test="heading-display">What is Hacktoberfest?</h2>
```

Comment:
⛏️ Maybe go with the old version here.

Reply:
I removed the question mark, to match previous version.  Let me know if I misunderstood.  
Commit:  4f5d0c2f70b91c4f8f4190a7289064862eac6bfc